packages/dux-code-report/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "dux-code-report",
+  "version": "0.0.0",
+  "description": "Creates a report in prep for a DUX Review",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC"
```

Comment:
Was there a reason to specify an ISC licence? Our other packages use apache, and the code header in your `index.js` claims it's covered by an apache licence...

Reply:
Just the default, will correct